pkg/cmd/util_test.go

```diff
@@ -188,6 +188,32 @@ func TestSubstituteVariables(t *testing.T) {
 			[]byte(""),
 			true,
 		},
+		{
+			[]byte(`
+				name: httpd
+				containers:
+				- image: foo/bar:[[ TEST_IMAGE_TAG:latest ]]
```

Comment:
can we have tests for following cases?:

`[[ TEST_IMAGE_TAG: latest ]]`
`[[ TEST_IMAGE_TAG: latest]]`
`[[TEST_IMAGE_TAG:latest]]`

Reply:
sure

Reply:
@kadel , added test for above cases